pkgs/os-specific/linux/kernel/common-config.nix

```
@@ -521,6 +521,15 @@ let
       X86_SGX     = whenAtLeast "5.11" yes;
       # Allow KVM guests to load SGX enclaves
       X86_SGX_KVM = whenAtLeast "5.13" yes;
+
+      # AMD Cryptographic Coprocessor (CCP)
+      CRYPTO_DEV_CCP  = whenAtLeast "4.14" yes;
+      # AMD SME
+      AMD_MEM_ENCRYPT = whenAtLeast "4.14" yes;
```

After #201845 this maybe should be:

```suggestion
      # AMD Cryptographic Coprocessor (CCP)
      CRYPTO_DEV_CCP  = yes;
      # AMD SME
      AMD_MEM_ENCRYPT = yes;
```

---

Fixed: https://github.com/NixOS/nixpkgs/compare/c6ee306837ae7ab651cf38fc3b44177f2af853db..c0ae4817572173a4b4c45899d5d6e5a9c7c01d73